spark/agent/README.md

```diff
@@ -0,0 +1,100 @@
+# Spark Agent / Harvester
+
+This module is responsible for listening to spark command events and converting them to spline lineage.
+
+## Spark commands
+Some events provided by Spark are not yet implemented. Some of them will be implemented in future 
+and some of them bear no lineage information and should be ignored.
+
+All commands inherits from ```org.apache.spark.sql.catalyst.plans.logical.Command```.
+
+You can see how to produce unimplemented commands in ```za.co.absa.spline.harvester.SparkUnimplementedCommandsSpec```.
+### Implemented
+```
+CreateDataSourceTableAsSelectCommand (org.apache.spark.sql.execution.command)
```

Comment:
why not use plain `-` (\<ul>)?

Reply:
I usually use ``` to mark that the word is a piece of code or specific Scala class. Here it's all classes, so following the same logic I just put them all in one block.

Reply:
Yes, I agree 100% on marking code with ` or ``` depending on the situation. But I would expect this.

- `CreateHiveTableAsSelectCommand` (org.apache.spark.sql.hive.execution)
- `CreateTableCommand` (org.apache.spark.sql.execution.command)


In no way do I want to force you into it. Just give it a thought